docs/source/model_design/vivarium_features/other_causes_ylds/index.rst

```diff
@@ -0,0 +1,230 @@
+..
+  Section title decorators for this document:
+  
+  ==============
+  Document Title
+  ==============
+  Section Level 1
+  ---------------
+  Section Level 2
+  +++++++++++++++
+  Section Level 3
+  ~~~~~~~~~~~~~~~
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _other_causes_ylds:
+
+.. role:: underline
+    :class: underline
+
+=========================================================
+YLDs due to other causes
+=========================================================
+
+Years lived with disability (YLDs) are an important measure of health loss and 
+contribute to estimates of disability adjusted life years (DALYs) in our model. 
+This document discusses how YLDs are tracked in vivarium by default, how that 
+compares to GBD methodology, and discusses potential improvements to this 
+default behavior.
+
+.. contents::
+   :local:
+   :depth: 2
+
+Background and Motivation
+--------------------------
+
+GBD methodology
+++++++++++++++++
+
+Disability weight estimation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The GBD study relies on disability weights (DWs) to estimate years lived with 
+disability (YLDs). DWs represent the magnitude of health loss associated with a 
+specific health outcome and are measured on a scale from 0 to 1; 0 implies a 
+state equivalent to full health and 1 equivalent to death.
+
+Disability weights have been estimated through a series of surveys, described 
+in detail in the GBD methods appendix along with a complete listing of 
+disability weight values for all of the health states included in the GBD study.
+
+Comorbidity correction
+~~~~~~~~~~~~~~~~~~~~~~
+
+The disability weights estimated through the process described above consider
+each health state individually and must be corrected for the simultaneous 
+presence of multiple disability-causing health states within a given 
+individual. The GBD study performs this comorbidity correct (termed "COMO") 
```

Comment:
```suggestion
individual. The GBD study performs this comorbidity correction (termed "COMO") 
```
This probably makes no difference but this version feels less jargony to me somehow?